docs/3-understanding-dataLoader.md

```diff
@@ -62,19 +62,19 @@ The GraphQL execution engine will always try to execute fields in parallel in or
     using HotChocolate.Types;
 
     namespace ConferencePlanner.GraphQL
+    
+    public static class ObjectFieldDescriptorExtensions
     {
-        public static class ObjectFieldDescriptorExtensions
+        public static IObjectFieldDescriptor UseDbContext<TDbContext>(
```

Comment:
This all can also be simplified.... do not worry about it ... I will merge your PR as is and chime in a bit to simplify the section.